src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1826,59 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+
+    public function updateVerificationStatus($userToken,$verificationStatus){
+        if (empty($userToken)) {
+            throw new HyperwalletArgumentException('userToken is required!');
+        }
+        if (empty($verificationStatus)) {
```

Comment:
Need to add UT cases for the below cases: 
empty($verificationStatus
$verificationStatus != User::VERIFICATION_STATUS_REQUESTED
!in_array($fromValidationStatus,$expectedFromValues

Reply:
Its already there related IT method is testRequestUserVerificationWithInvalidFromVerificationStatusIT

Reply:
Requested for UT cases and not IT.

Reply:
Done